### PR TITLE
[es] Deep learning

### PR DESCRIPTION
**Pull request by fdiazgon**
Some notes:

* For reference, I am from Spain. I tried not to use slang words that you can only hear here. If some fellow Latin American think something could be expressed clearer, please, let me know.

* I have tried to be faithful to the original text. However, sometimes I needed to change some words. E.g.,
- ... are a class of models that are built with layers.
- ... are a class of models that are organized with layers. (since "built" sounds weird in Spanish).

* I had to look for the translation of "bias". It seems that even in Spanish literature, the English term is commonly used.

* I think it is not worth to translate some concepts like (e.g., Cross-Entropy, Leaky ReLU). Sometimes the English term is always used.

* "Loss" is very hard to translate. I didn't want to use "cost" because, usually, we speak about cost when we compute the loss for all the training examples. I have used "error" instead. (However, for some people, "cost function" and "loss function" are synonyms).

### CODE REVIEW

**Comment by shervinea** [comment]
Thank you so much for being so meticulous in your work Fernando! 

It seems that there is already another translation of this cheatsheet [here](https://github.com/shervinea/cheatsheet-translation/pull/22): could you please take a look at it? I am marking this pull request as `duplicate` so that we can all focus on a single one.

**Comment by fdiazgon** [comment]
Yes, my bad. I saw that there were no translations for this one but I forgot to check the pull requests.
